### ISSUE TITLE
Please remove official Brave Logos from the artwork

### ISSUE THREAD

**Issue by bradleyrichter** [KudoAI/bravegpt#9]
![image](https://github.com/KudoAI/bravegpt/assets/13509546/d2dcf0d0-d299-45e6-8933-e9264e8aca2d)
 
We can't allow this [logo usage] simply because it looks like Brave Software created it. Totally fine to create a plugin for Brave Search specifically as you have done. Just make it clear that it's from you and not from us please.

Otherwise, go for it. Let us know if you have any questions. 

(Fine print: devs or someone other than me may have other thoughts. I am just offering branding help.)

**Comment by adamlui** [comment]
Since when was making demands "offering help"? Have you even used this script? In no way would _anyone_ ever be confused to think your employer is the author (it clearly says KudoAI: )
![image](https://github.com/KudoAI/bravegpt/assets/10906554/177bf373-8713-415f-93f3-dc9c2ec986dd)

This repo is literally called KudoAI/bravegpt also, not brave/bravegpt. It does **not** "look like Brave Software created it."

And considering its existence increases traffic to your employer's product (not to mention Brendan actively stalks me already, for example when he found about about my past cultofgeek.com thru Brave Ads, he launched an online store himself, and when this script came out, he suddenly pivoted to focus harder on search and rug pull the devout BAT userbase) a little gratitude would be appreciated.

TL;DR thank me, fuck off or send a cease & desist

**Comment by bradleyrichter** [comment]
Sorry I wasn't clear. I meant "this" as in specifically "the brave logo with GPT on the end " as I pasted above. That's all.

**Comment by adamlui** [comment]
Sorry I wasn't clear, ask your employer's legal dept to produce a cease & desist & send to my email adam@kudoai.com if you are insistent on harassing a contributor to your employer's revenue (thus even your own salary) and has been so reasonable in explaining no one in their right mind would be confused (this logo only appears on a readme whose URL is KudoAI/bravegpt) that any continued coercion becomes undue harassment instead, thus I will not be making it easier for you without at least other departments finding out what you're doing out here on GitHub Mr. Brad.

**Comment by bsclifton** [comment]
@adamlui you can find the cease and desist here:
[https://brave.com/legal/bravegpt-cease-and-desist](https://www.youtube.com/watch?v=dQw4w9WgXcQ)
